### ISSUE TITLE
Read Twitter user's media tab, grab last dozen pics, read alt text/run OCR, attempt to parse-out a bsky address

### ISSUE THREAD

**Issue by DrAndiLowe** [kawamataryo/sky-follower-bridge#162]
For profiles that haven't got a match on bsky using the existing methods:

Lots of people on Twitter only dump a screencap of their bsky profile page on their feed. This makes it very difficult to match them to their bsky profile. Sadly, it's a common practise. It would be cool if there was a button I could click on to activate "deep search" more for individual profiles without a match from username/displayname matching. In the first instance, one could retrieve that last few pics on a user's media tab and parse any alt text for a bsky profile address. If this doesn't work, OCR could be an option.

Depends on #68
